source/client/process_impl.cc

```diff
@@ -522,7 +523,8 @@ bool ProcessImpl::runInternal(OutputCollector& collector, const std::vector<UriP
         admin_, Envoy::Runtime::LoaderSingleton::get(), store_root_, tls_,
         dispatcher_->createDnsResolver({}, dns_resolver_options), *ssl_context_manager_,
         *dispatcher_, *local_info_, secret_manager_, validation_context_, *api_, http_context_,
-        grpc_context_, router_context_, access_log_manager_, *singleton_manager_, envoy_options);
+        grpc_context_, router_context_, access_log_manager_, *singleton_manager_, envoy_options,
+        quic_stat_names_);
```

Comment:
rather than storing quic_stat_names_ on the object (requiring changes here and in the .h file), could we instead just use store_root_.symbolTable() directly here?
As is, I believe we are essentially storing this on our ProcessImpl twice.

Reply:
discussed this offline. the new QuicStatNames argument in the ClusterManagerFactory constructor is a non const reference. so it expects an l-value to be passed in.